### PR TITLE
Option for multiple trip fare and trip interruptions

### PR DESCRIPTION
**Pull request by CGantert345**
options to define some domestic fares with:
 - allowed interruptions
 - multiple trip fare  
 in more detail

### CODE REVIEW

File: specification/v1.4.0-rc/OSDM-offline-model-v1.4.0-rc.json [hunk 1]
```diff
@@ -1,2246 +1,2362 @@
...
+			]
+		},
+		"TripAllocationProcess": {
+			"description": "NONE: Individual trips don't need to be allocated. MANUAL: The passenger has to allocate a trip manually (This should not be used as it is by desiugn non-interoperable). ACTIVATION: Individual trips can be allocatted by activation of the ticket. FULFILLMENT A new fulfillment needs to be created to allocate a trip",
+			"type": "string",
+			"x-extensible-enum": [
+				"NONE",
+				"MANUAL",
+				"ACTIVATION",
+				"FULFILLMENT"
```
Comment (ConscritNeuneu) [inline specification/v1.4.0-rc/OSDM-offline-model-v1.4.0-rc.json]: Do we have a position on US English vs UK English? The version with to `l` is american English. https://en.wiktionary.org/wiki/fulfilment

File: specification/v1.4.0-rc/OSDM-offline-model-v1.4.0-rc.json [hunk 2]
```diff
@@ -1,2246 +1,2362 @@
...
+				"tripInterruptionConstraint": {
+					"$ref": "#/definitions/TripInterruptionConstraintDef"
+				}
+			},
+			"required": [
+				"validityRange"
+			],
+			"additionalProperties": false
+		},
+		"TripInterruptionProcess": {
```
Comment (ConscritNeuneu) [inline specification/v1.4.0-rc/OSDM-offline-model-v1.4.0-rc.json]: Is there a definition somewhere of what constitutes a trip interruption?

**Review by schlpbch — CHANGES_REQUESTED** [review]
Diff to large :-(

File: specification/v1.4.0-rc/OSDM-offline-model-v1.4.0-rc.json (hunk 1 above)
Comment (schlpbch) [reply specification/v1.4.0-rc/OSDM-offline-model-v1.4.0-rc.json]: I'm using the CodeSpellChecker plugin in VSCode set to US-English

**Comment by ConscritNeuneu** [comment]
> Diff to large :-(

You can use GitHub"s "ignore white space". Most of the diff is a change in the json indentation
image

File: specification/v1.4.0-rc/OSDM-offline-model-v1.4.0-rc.json (hunk 2 above)
Comment (jspetrak) [reply specification/v1.4.0-rc/OSDM-offline-model-v1.4.0-rc.json]: Not sure about UIC-endorsed regularions but in Czechia, Ministry of Transport and Ministry of Finance both issue generally required conditions of carriage, and these set the conditions for interruption, e.g. each opet ticket can be interrupted after travelling for 100 or more kilometers. And rest of journey must be completed within ticket time validity which is 0.00 on travel date till 6.00 on the following day (and the validity is also regulated).

File: specification/v1.4.0-rc/OSDM-offline-model-v1.4.0-rc.json (hunk 1 above)
Comment (jspetrak) [reply specification/v1.4.0-rc/OSDM-offline-model-v1.4.0-rc.json]: Good questions. en_GB would at least ensure I don't need to double check each time if double L is written after first or second F. :) And I was also corrected once incorrectly that the correct spelling is fullfiled. :/

File: specification/v1.4.0-rc/OSDM-offline-model-v1.4.0-rc.json (hunk 2 above)
Comment (CGantert345) [reply specification/v1.4.0-rc/OSDM-offline-model-v1.4.0-rc.json]: Local goverments regulate sometimes the domestic tickets and sometimes only the domestic full fare tickets, international tariffs are not covered by these regulations, so the fare providers define them on their own.

Comment (jspetrak) [reply specification/v1.4.0-rc/OSDM-offline-model-v1.4.0-rc.json]: Of course, regional networks are one different world. Internation fares usually have specific rules as well (train-bound fixed fares, flexible fares, ...) I guest we should not try to express everything in OSDM like we did in refundability of fares - yes, no, with conditions (but details not expressed in the API response).

**Review by schlpbch — APPROVED** [review]
Peer reviewed with Clemens

**Pull request merged**